pkg/build/types/parser/deffile.go

```diff
@@ -200,7 +200,7 @@ func populateDefinition(sections map[string]string, d *types.Definition) (err er
 	// Files are parsed as a map[string]string
 	filesSections := strings.TrimSpace(sections["files"])
 	subs := strings.Split(filesSections, "\n")
-	var files []types.FileTransport
+	files := make([]types.FileTransport, 0, len(subs))
```

Comment:
Is CI failure related to this change ?

Reply:
good question, looking

Reply:
I fixed this in the most recent change, thanks!